app/service/parsing_svc.py

```diff
@@ -21,14 +55,11 @@ def __init__(self):
         for result in [r for r in results if not r['parsed']]:
             parser = await data_svc.get('core_parser', dict(ability=result['link']['ability']))
             if parser and result['link']['status'] == 0:
-                if parser[0]['name'] == 'json':
-                    matched_facts = parsers.json(parser[0], b64decode(result['output']).decode('utf-8'), self.log)
-                elif parser[0]['name'] == 'line':
-                    matched_facts = parsers.line(parser[0], b64decode(result['output']).decode('utf-8'), self.log)
-                elif parser[0]['name'] == 'mimikatz':
-                    matched_facts = mimikatz_parser.mimikatz(b64decode(result['output']).decode('utf-8'), self.log)
-                else:
-                    matched_facts = parsers.regex(parser[0], b64decode(result['output']).decode('utf-8'), self.log)
+                if parser[0]['name'] in self.parsers:
+                    matched_facts = self.parsers[parser[0]['name']].parse(parser[0], b64decode(result['output']).decode('utf-8'))
+                else:                
+                    self.log.error('Unable to find parser %s' % parser[0]['name'])
+                    matched_facts = []
 
                 source = (await data_svc.explode_sources(dict(name=operation['name'])))[0]
                 for match in matched_facts:
```

Comment:
what submodule changes are you pulling in? (below)

Reply:
Crap, that was unintended.  Not well-versed in git submodules, how do you recommend I resolve this?